README.md

```diff
@@ -6,6 +6,8 @@
 
 Notation is a CLI project to add signatures as standard items in the registry ecosystem, and to build a set of simple tooling for signing and verifying these signatures. This should be viewed as similar security to checking git commit signatures, although the signatures are generic and can be used for additional purposes. Notation is an implementation of the [Notary project specifications][notaryv2-specs].
 
+> **IMPORTANT:** The documentation for using `notation` is available [here](https://notaryproject.dev/docs/).
```

Comment:
How about
```suggestion
> **Note** The documentation for using `notation` is available [here](https://notaryproject.dev/docs/).
```

It renders to
> **Note** The documentation for using `notation` is available [here](https://notaryproject.dev/docs/).

Reply:
thanks @shizhMSFT

Incorporated your feedback